crates/node-maintainer/src/maintainer.rs

```diff
@@ -222,6 +232,16 @@ impl NodeMaintainer {
                         let dir_clone = target_dir.clone();
                         async_std::task::spawn_blocking(move || temp.extract_to_dir(&target_dir))
                             .await?;
+                        pb.inc(1);
+                        pb.set_message(format!(
+                            "{}@{}",
+                            me.graph[child_idx].package.name(),
```

Comment:
you should use `{:?}` here with `me.graph[child_idx].resolved`, because [`PackageResolution` has a `Debug` implementation for this use case](https://github.com/orogene/orogene/blob/main/crates/nassun/src/resolver.rs#L41-L55)

Reply:
Cool, done here:
https://github.com/orogene/orogene/pull/145/commits/8a3227d5192775383292d1ec244b4f336149c8ec